content/en/docs/test.md

```diff
@@ -369,9 +369,13 @@ Notes catch the reader's attention without a sense of urgency.
 
 You can have multiple paragraphs and block-level elements inside an admonition.
 
-| Or | a | table |
+| Or | | Even |
```

Comment:
```suggestion
|  Or |  a  | table |
```

Reply:
Please see @Sea-n comment above.

Reply:
It is not necessary to have a table body to make it render as a table the changes I suggested would make it render as a table too, any of the approaches should work

Reply:
Yes, but rendering only the header, could be sometimes misleading to the user because it's not very clear that it is a table.

Reply:
Gotcha 👍